
 This patch can be applied to version 3.0.1 and should
 fix bug #1309 btape fill multiple tape generates error but
 end of fill test says successful.

 Apply it to version 3.0.1 with:

 cd <bacula-source>
 patch -p0 <3.0.1-btape.patch
 ./configure <your-options>
 make
 ...
 make install



Index: src/stored/btape.c
===================================================================
--- src/stored/btape.c	(revision 8902)
+++ src/stored/btape.c	(working copy)
@@ -414,7 +414,6 @@
       }
    }
    dev->rewind(dcr);
-   dev->weof(1);
    write_new_volume_label_to_dev(dcr, cmd, "Default", false,/*no relabel*/ true /* label dvd now */);
    Pmsg1(-1, _("Wrote Volume label for volume \"%s\".\n"), cmd);
 }
